.github/workflows/integration.yml

```diff
@@ -34,22 +34,13 @@ jobs:
         working-directory: ./ops
         run: ./scripts/build-ci.sh
 
-      - name: Bring the stack up and wait for the sequencer to be ready
+      - name: Bring the stack up
         working-directory: ./ops
-        run: docker-compose up -d && ./scripts/wait-for-sequencer.sh
+        run: docker-compose up -d
```

Comment:
Won't this bring up the integration tests? It would conflict with the next step

Reply:
No, there's a `scale: 0` in the docker-compose

Reply:
Didn't notice that at first, thanks for pointing out